src/routes/sig.py

```
@@ -176,11 +180,36 @@ def get_sig_tags(
     return SigTagResponse.model_validate_list(sig_service.get_sig_tags(id))
 
 
-@sig_router.delete("/sig/{id}/tag/{label}", status_code=204)
+@sig_router.delete("/sig/{id}/tag/{tag_id}", status_code=204)
 def remove_sig_tag(
     id: int,
```

Ruff (0.15.5)

[error] Function argument `id` is shadowing a Python builtin

(A002)